packages/caliper-core/lib/manager/report/template/report.html

```diff
@@ -122,7 +122,7 @@
 <body style="font-family: IBM Plex Sans; font-weight: 200;">
     <main>
         <div class="left-column">
-            <img src="https://hyperledger-caliper.github.io/caliper/assets/img/hyperledger_caliper_logo_color.png" style="width:95%;" alt="">
+            <img src="/assets/hyperledger_caliper_logo_color.png" style="width:95%;" alt="">
```

Comment:
Unfortunately at the moment I don't think this would work. We need to consider how uses of caliper say via npm modules would work, plus if they publish the report somwhere they would need to be able to get the file from somewhere. Not sure what the best solution is to this at this point.